pkg/backends/otlp/config.go

```diff
@@ -40,28 +42,30 @@ type Config struct {
 	UserAgent string `mapstructure:"user_agent"`
 }
 
-func newDefaultConfig() Config {
-	return Config{
-		Transport:           "default",
-		MaxRequests:         runtime.NumCPU() * 2,
-		TimerConversion:     "AsGauge",
-		HistogramConversion: "AsGauge",
-		UserAgent:           "gostatsd",
+func newDefaultConfig() *Config {
+	return &Config{
+		Transport:   "default",
+		MaxRequests: runtime.NumCPU() * 2,
```

Comment:
Is this too conservative?

Reply:
So it really depends on the backend supporting out of order writes, if it does, you can open this up wide, otherwise, restricting it is a good idea.